website_and_docs/content/documentation/webdriver/troubleshooting/logging.en.md

```diff
@@ -9,70 +9,211 @@ description: >
-Selenium::WebDriver.logger.ignore(:deprecations)
-```
-
-**INFO**
-
-This is where the most useful information gets logged. Selenium logs the endpoints and payloads
-sent to and received from the driver or server. This is a great way to see what Selenium is actually
-doing under the hood, and can be used to determine if it is Selenium code or driver code that
-is causing a problem. (Unfortunately, we can't blame the driver if Selenium is sending incorrect syntax).
-
-**DEBUG**
-
-This is less useful information where we log things about the servers and the sockets, and header information, etc.
-Debug mode is set if either `$DEBUG` is true or `ENV['DEBUG']` has a value.
-
------
 
-{{< alert-content >}}
-Descriptions of how to set and use logging in Java, Python, JavaScript, and .NET
-{{< /alert-content >}}
+{{< tabpane langEqualsHeader=true text=true >}}
+  {{< tab header="Java" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Python" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="CSharp" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Ruby" >}}
+  <p>Ruby logs deprecations as warnings, specifying what is changing, what needs to be
+  used instead. It may include additional messages, and always includes an ID.</p>
+
+  <p>For example:</p>
+  <blockquote>2022-12-24 16:07:09 WARN Selenium [DEPRECATION] [:jwp_caps] \`Capabilities#version=\` is deprecated. Use \`Capabilities#browser_version=\` instead.</blockquote>
+
+  <p>Because these items can get annoying, we've provided an easy way to turn them off.</p>
+
+  <p>To turn off a specific warning, set the ID to ignore:</p>
+  <pre>
+  Selenium::WebDriver.logger.ignore(:jwp_caps)</pre>
+  <br>
+  <p>It accepts an `Array` to turn off multiple IDs:</p>
+  <pre>
+  Selenium::WebDriver.logger.ignore(%i[jwp_caps pause pauses])</pre>
+  <br>
+  <p>To turn off all *deprecation* notices:</p>
+  <pre>
+  Selenium::WebDriver.logger.ignore(:deprecations)</pre>
+  {{< /tab >}}
+  {{< tab header="JavaScript" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Kotlin" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+{{< /tabpane >}}
+
+**Useful Information**
+
+This is the default level where Selenium logs things that users should be aware of but do not need to take actions on.
+It presents information such as requests and responses between driver and server, payload, etc.
+
+Different languages have different level to log information.
+
+{{< tabpane langEqualsHeader=true text=true >}}
+  {{< tab header="Java" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Python" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="CSharp" >}}
+  {{< alert-content >}}{{< /alert-content >}}
+  {{< /tab >}}
+  {{< tab header="Ruby" >}}
+  Logs useful information at level: <code>:info</code>
```

Comment:
```suggestion
Because the Ruby `Logger` class only has one "debug" level, Selenium is currently using the `:info` level as a general debug mode, and `:debug` as a lower level debug mode.
To bring things in line with other languages, we are considering [changing this behavior]({{< ref "https://github.com/SeleniumHQ/selenium/issues/11797" >}}).
For now, both info and warnings are handled at the default `:warn` level.
```